sycl/test/on-device/README.md

```diff
@@ -11,3 +11,59 @@ It is developer responsibility to move the tests from this directory to
 [DPC++ E2E test suite](https://github.com/intel/llvm-test-suite/tree/intel/SYCL)
 or [KhronosGroup/SYCL-CTS](https://github.com/KhronosGroup/SYCL-CTS) once the
 feature is stabilized.
+
+# Running tests on deployed DPC++ compiler and runtime
+The tests in this directory can be run on deployed DPC++ compiler and runtime.
+It is expected that get_device_count_by_type tool is prebuilt by the developer
+and tools with full path os provided as value for GET_DEVICE_TOOL LIT
+parameter. 
+
+```bash
+# clone llvm_project repo
+git clone https://github.com/intel/llvm sycl_lit
+cd sycl_lit
+export ROOT=`pwd`
+mkdir build
+cd build
+
+# set extra environment
+export EXTCMPLRROOT=# path to deployed SYCL compiler and runtime
+export GET_DEVICE_TOOL=# Path to get full path to utility detecting available devices ([source](../../tools/get_device_count_by_type.cpp))
+
+
+# Configure project
+cmake -DLLVM_TARGETS_TO_BUILD="X86" -DLLVM_EXTERNAL_PROJECTS=sycl-test \
+      -DLLVM_EXTERNAL_SYCL_TEST_SOURCE_DIR=$ROOT/sycl/test/on-device \
+      -DSYCL_SOURCE_DIR=$ROOT/sycl -DOpenCL_LIBRARIES=$EXTCMPLRROOT/lib \
+      $ROOT/llvm
+
+# Build LIT tools
+make FileCheck not llvm-config
+
+# Run tests for OpenCL BE
+/usr/bin/python3.6 $ROOT/llvm/utils/lit/lit.py -v \
```

Comment:
Is it necessary to hardcode path to the python and it's version?

Reply:
fixed